src/sidebar/services/toast-messenger.ts

```diff
@@ -53,14 +59,16 @@ export class ToastMessengerService {
    * extant in the store's collection of toast messages (i.e. has the same
    * `type` and `message` text of an existing message).
    *
-   * @param {('error'|'success'|'notice')} type
-   * @param {string} messageText - The message to be rendered
-   * @param {MessageOptions} options
+   * @param messageText - The message to be rendered
```

Comment:
I think you could remove this `@param` comment; I'm not convinced it adds much.

Reply:
Agree